about/timescaledb-editions.md

```diff
@@ -60,11 +60,8 @@ to repair" and make modifications to the source code and run it in their own
 on-premises or cloud infrastructure. However, you cannot make modifications to
 the TimescaleDB Community Edition source code and offer it as a service.
 
-Timescale provides hosted versions of TimescaleDB Community Edition:
-[Timescale][timescale-cloud] and
-[Managed Service for TimescaleDB][mst]. Timescale is a cloud-native
-platform for time-series, hosted in AWS. MST is a database-as-a-service offering
-for TimescaleDB. MST is offered in AWS, Azure, and Google Cloud.
+You can access a hosted version of TimescaleDB Community Edition through
+[Timescale][timescale-cloud], which is a cloud-native platform for time-series.
```

Comment:
I am not sure. I always thought that Timescale Cloud was not the hosted version of the TimescaleDB community.

Reply:
The original seemed to say both Cloud and MST were hosted versions of Community. Maybe @mfreed can confirm?

Reply:
This is correct